dipy/data/fetcher.py

```diff
@@ -691,6 +691,15 @@ def fetcher():
      'c1499005baccfab865ce38368d7a4c7f'],
     doc="Download Bundle Warp dataset")
 
+fetch_wigner_Jmat = _make_fetcher(
```

Comment:
`jmat` lowercase to avoid any confusion and respect the standard of the other fetcher